### PR TITLE
Log customization (Progress)

### PR DESCRIPTION
**Pull request by lamtu1**
Adding in the development of customizing the log output using Logger. Currently still in progress.

### CODE REVIEW

**Comment by lamtu1** [comment]
Screenshot 2022-11-18 at 3 33 34 PM

Progress so far using ```Logger``` on customization.

**Comment by lamtu1** [comment]
Screenshot 2022-11-23 at 10 34 51 AM
New progress image of the log

**Comment by lamtu1** [comment]
Screenshot 2022-11-23 at 2 10 17 PM
New screenshot with the color escape key remove.

**Comment by lamtu1** [comment]
Screenshot 2022-11-23 at 2 35 19 PM
Removing the bracket from the string field area of ```req_id``` & ```req_uri```

File: config/environment.rb [hunk 1]
```diff
@@ -3,3 +3,38 @@
 
 # Initialize the Rails application.
 Rails.application.initialize!
+
+# Adding in hierarchy of config for PROD, DEV, and STAG
+Rails.application.configure do
```
Comment (CGillen) [inline config/environment.rb:8]: Tab => 2 space

File: config/environments/production.rb [hunk 2]
```diff
@@ -90,7 +90,7 @@
   config.public_file_server.enabled = true
 
   # Use default logging formatter so that PID and timestamp are not suppressed.
-  config.log_formatter = ::Logger::Formatter.new
+  # config.log_formatter = ::Logger::Formatter.new
```
Comment (CGillen) [inline config/environments/production.rb:93]: Just remove both line, no comments

File: config/environments/staging.rb [hunk 3]
```diff
@@ -50,7 +50,7 @@
   config.log_level = :debug
 
   # Prepend all log lines with the following tags.
-  config.log_tags = [:request_id]
+  # config.log_tags = [:request_id]
```
Comment (CGillen) [inline config/environments/staging.rb:53]: No comments

File: config/environments/staging.rb [hunk 4]
```diff
@@ -92,7 +92,7 @@
   config.public_file_server.enabled = true
 
   # Use default logging formatter so that PID and timestamp are not suppressed.
-  config.log_formatter = ::Logger::Formatter.new
+  # config.log_formatter = ::Logger::Formatter.new
```
Comment (CGillen) [inline config/environments/staging.rb:95]: No comments

File: config/initializers/lograge.rb [hunk 5]
```diff
@@ -3,13 +3,7 @@
 if %w[development test].include? Rails.env
   Rails.application.configure do
     config.lograge.enabled = true
-    config.lograge.custom_options = lambda do |event|
```
Comment (CGillen) [inline config/initializers/lograge.rb:6]: Reset this file and open a new ticket for removing LogRage. In it ping Ryan O about if we're using any existing LR functionality

**Review by CGillen — CHANGES_REQUESTED** [review]

**Review by CGillen — APPROVED** [review]

**Pull request merged**